base/reflection.jl

```diff
@@ -2378,6 +2378,7 @@ Determine the module containing the (first) definition of a generic
 function.
 """
 parentmodule(f::Function) = parentmodule(typeof(f))
+parentmodule(f::Core.IntrinsicFunction) = Core.Intrinsics
```

Comment:
I think this is the wrong fix (since it won't change the result of ``parentmodule(Core.IntrinsicFunction)`. The better fix would be for `IntrinsicFunction.name.module` to be set correctly. @Keno, do you know why it isn't?